### PR TITLE
fix: improve Kafka Prometheus metrics docs

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes

- Converted the metrics list from a bullet list to a table format for a cleaner and more organized display.
- Added missing descriptions for several metrics to provide clearer insights.
- Improved the overall documentation for Kafka Prometheus metrics. [DOC-617](https://aiven.atlassian.net/browse/DOC-617).
- Included new metrics specific to tiered storage.

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

[DOC-617]: https://aiven.atlassian.net/browse/DOC-617?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/products/kafka/reference/kafka-metrics-prometheus.md [hunk 1]
```diff
@@ -2,518 +2,549 @@
...
-
-Metrics below with
-`kafka_controller_ControllerStats_LeaderElectionRateAndTimeMs_`
-(Max/Mean/Min/StdDev) provide statistical measures about the leader
-election times.
-
-Metrics below with `kafka_controller_KafkaController_Metrics` provide
-insights into the state of the Kafka controller, like the number of
-active brokers, offline partitions, replicas to delete, etc.
+To calculate the rate of change for these `_count` metrics, you can use
```
Comment (giuseppelillo) [inline docs/products/kafka/reference/kafka-metrics-prometheus.md:147]:
```suggestion
To calculate the rate of change for these `_Count` metrics, you can use
```

**Review by giuseppelillo — APPROVED** [review]
LGTM, just left a small comment

File: docs/products/kafka/reference/kafka-metrics-prometheus.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/kafka/reference/kafka-metrics-prometheus.md]: Fixed

File: docs/products/kafka/reference/kafka-metrics-prometheus.md [hunk 2]
```diff
@@ -2,518 +2,546 @@
...
-:::tip
-You can check how to use Prometheus with Aiven in the
-[dedicated document](/docs/platform/howto/integrations/prometheus-metrics).
-:::
+Explore common metrics available via Prometheus for your Aiven for Apache Kafka® service.
+
+## How to retrieve metrics
+
+To get the complete list of available metrics for your service, query the
+Prometheus endpoint. Follow these steps to get started:
```
Comment (wojcik-dorota) [inline docs/products/kafka/reference/kafka-metrics-prometheus.md:10]: Or shorter
```suggestion
Prometheus endpoint. To get started:
```

Comment (wojcik-dorota) [inline docs/products/kafka/reference/kafka-metrics-prometheus.md:10]: Or shorter
```suggestion
Prometheus endpoint. To get started:
```

File: docs/products/kafka/reference/kafka-metrics-prometheus.md [hunk 3]
```diff
@@ -2,518 +2,546 @@
...
+
+1. Ensure you have the following information:
+
+   - Aiven project certificate: `ca.pem`. To download the CA certificate, see
+     [Download CA certificates](/docs/platform/concepts/tls-ssl-certificates#download-ca-certificates).
+   - Prometheus credentials: `<PROMETHEUS_USER>:<PROMETHEUS_PASSWORD>`
+   - Aiven for Apache Kafka hostname: `<KAFKA_HOSTNAME>`
+   - Prometheus port: `<PROMETHEUS_PORT>`
+
+1. Use the curl command to query the Prometheus endpoint:
```
Comment (wojcik-dorota) [inline docs/products/kafka/reference/kafka-metrics-prometheus.md:20]:
```suggestion
1. Use the `curl` command to query the Prometheus endpoint:
```

File: docs/products/kafka/reference/kafka-metrics-prometheus.md [hunk 4]
```diff
@@ -2,518 +2,546 @@
...
--   `system_n_cpus`: Number of CPU cores available.
--   `system_n_users`: Number of users logged in.
--   `system_uptime`: Time for which the system has been up and running.
+CPU utilization metrics offer insights into CPU usage. These metrics
+include time spent on different processes, system load, and overall uptime.
+
+| Metric                  | Description                                                                                           |
+|-------------------------|-------------------------------------------------------------------------------------------------------|
+| `cpu_usage_guest`       | CPU time spent running a virtual CPU for guest operating systems.                                     |
+| `cpu_usage_guest_nice`  | CPU time spent running low-priority virtual CPUs for guest operating systems. These processes can be interrupted by higher-priority tasks. Metric is measured in hundredths of a second. |
```
Comment (wojcik-dorota) [inline docs/products/kafka/reference/kafka-metrics-prometheus.md:39]: Except for this specific row, I'd remove all the final full stops from the `description` column coz they are not sentences. This comment refers to all the tables in this article wherever applicable.

File: docs/products/kafka/reference/kafka-metrics-prometheus.md [hunk 5]
```diff
@@ -2,518 +2,546 @@
...
-name.
+#### Cumulative counters (`_count`)
+Metrics with a `_count` suffix are cumulative counters. They track the total number of
+occurrences for a specific event since the broker started.
 
-To see the rate of change of these `_Count` metrics, you can apply a
-function such as the `rate()` function in PromQL.
-:::
+**Example:**
+- `kafka_controller_ControllerStats_LeaderElectionRateAndTimeMs_Count`: Total number of
```
Comment (wojcik-dorota) [inline docs/products/kafka/reference/kafka-metrics-prometheus.md:134]: For this example and the one below, I'd remove an unordered list coz there is one example only in each case.
```suggestion
`kafka_controller_ControllerStats_LeaderElectionRateAndTimeMs_Count`: Total number of
```

File: docs/products/kafka/reference/kafka-metrics-prometheus.md [hunk 6]
```diff
@@ -2,518 +2,546 @@
...
+### Tiered storage metrics
+
+Aiven for Apache Kafka includes several metrics to monitor the performance and
+health of your Apache Kafka broker's tiered storage operations. Access these metrics
+through Prometheus to gain insights into various aspects of tiered storage, including
+data copying, fetching, deleting, and their associated lags and errors.
+
+| Metric                                                           | Description                                                                                                      |
+|------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| `kafka_server_BrokerTopicMetrics_RemoteCopyBytesPerSec_Count`    | The number of bytes per second being copied to remote storage.                                                   |
```
Comment (wojcik-dorota) [inline docs/products/kafka/reference/kafka-metrics-prometheus.md:338]: Up to this point, you don't use an article (the/a/an) when starting the descriptions. I'd continue that way and wouldn't use articles as firsts words in table cells (which is mostly visual and practical thing I guess). This comment applies to all description cells in this article.

```suggestion
| `kafka_server_BrokerTopicMetrics_RemoteCopyBytesPerSec_Count`    | Number of bytes per second being copied to remote storage.                                                   |
```

**Review by wojcik-dorota — APPROVED** [review]
LGTM! A few ideas for possible improvements

**Pull request merged**